spec/Filter/Filter.BinaryLogic.Implementations.js

```diff
@@ -0,0 +1,17 @@
+describe('Filter.BinaryLogic', function() {
```

Comment:
More complex tests with defined operands are really needed, you just separately test that filters returns expected wrappers, but you don't test full filter use cases.